Sources/macro.d.ts

```diff
@@ -56,6 +56,18 @@ export enum TYPED_ARRAYS {
   Int32Array,
 }
 
+export type TypedArray =
+    | number[]
```

Comment:
@daker any reason why `number[]` is part of `TypedArray` ?

Reply:
i think you are right, it should not be part of it